src/views/Simulation.vue

```diff
@@ -48,6 +48,19 @@ export default {
     debug() {
       return this.$store.getters.getDebug
     },
+    showProgress() {
+      return ![
+        "recapitulatif",
+        "resultats",
+        "resultatsDetails",
+        "resultatsAttendus",
+      ].includes(this.$route.name)
+    },
+    currentProgressStyle() {
+      return {
+        width: `${this.$store.getters.progress * 100}%`,
+      }
+    },
```

Comment:
Je trouve que la formule de `this.$store.getters.progress` peut adaptée. Il faudrait se baser sur la `route` et pas sur les questions répondues.

Reply:
Done